packages/sls-mentor/src/rules/noDeprecatedRuntime/noDeprecatedRuntime.md

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
documentation files next to rules source code are legacy, they should now be put in apps/documentation/docs/rules. Just move this file everything else is fine
(our contributing documentation isn't up-to-date it will be fixed)